### ISSUE TITLE
CRUD Administration

### ISSUE THREAD

**Issue by TMM21042** [jaygoodswp/PHS#67]
An administration function is needed to see what is in the DB and tables.  And to interact with it (CRUD).  For example, it may be good to clean up the DB tables monthly to identify and remove demo registrants that have never returned or created multiple registrations from the same IP address, etc.  What is the right solution for an authorized administrator to interact with the backend of the PHS?

**Comment by jaygoodswp** [comment]
I agree @TMM21042 .  Usually I create an admin console available at /admin.  I would just need to a list of functions that you'll want to perform from an admin perspective.

**Comment by jaygoodswp** [comment]
Create admin for Users, Rules, and Sites
